service/src/main/java/com/ericsson/eiffel/remrem/generate/constants/RemremGenerateServiceConstants.java

```diff
@@ -35,4 +35,9 @@
         public static final String UNAVAILABLE_FOR_FAILIFNONEFOUND = "{\"status_code\": 406, \"result\": \"FAIL\", "
                 + "\"message\":\"No event id found with ERLookup properties\"}";
 
+        public static final String LIMIT = "The number of events to be displayed.";
```

Comment:
What do you mean with displayed?
Should it be something like this:
"Maximum number of events that will be looked up" or something. Or have I understood the LIMIT parameter wrong.
And maybe be more clear in the LIMIT name what it is intend to be used for, e.g.: EVENTS_LOOKUP_LIMIT.

Reply:
emil backmark suggested to change the parameter names like below
limit  to lookupLimit 
Done changes and pushed it, could you please check once and suggest it if its required

Reply:
ok

Reply:
Parameter name aside the description still doesn't help the user that much. "Displayed" doesn't make any sense at all, but even Tobias's suggestion leaves me wondering what happens if an ER query returns events beyond the limit. Will that fail the request? Or will only the first _N_ events be included?

Reply:
I believe that only the first _N_ events will be included. @xsudban, please correct me if I'm wrong. If I'm right I propose the description should be changed to:
"The maximum number of events returned from a lookup. If more events are found they will be disregarded. The order of the events is undefined, which means that what events are disregarded is also undefined."

Reply:
yes,  your right Emil, it will fetch only first N events only, 
Changed the description and pushed the changes also